source/Core/Inc/gui.hpp

```diff
@@ -21,7 +21,9 @@
 
 // Struct for holding the function pointers and descriptions
 typedef struct {
-  const char *description;
+  // The settings description index, please use the `SETTINGS_DESC` macro with
+  // the `SettingsItemIndex` enum. Use 0 for no description.
+  uint8_t description;

```

Comment:
Should this be a const?

Reply:
I don't think this really matters as a field. The `menuitem` arrays are already declared `const`.